test/tcp-connect.js

```diff
@@ -39,4 +39,42 @@ test('TCP connect works (echo test)', function (t) {
 
     server.listen(port)
   })
-})
\ No newline at end of file
+})
+
+test('TCP connect works with direct API (echo test)', function (t) {
+  portfinder.getPort(function (err, port) {
+    t.error(err, 'Found free port')
+    var child
+
+    var server = net.createServer()
+
+    server.on('listening', function () {
+      var env = { PORT: port }
+      helper.browserify('tcp-connect-direct.js', env, function (err) {
```

Comment:
This is copy/pasted from the test above, with the only difference being the js file it loads here ^^